src/parser/variable.js

```diff
@@ -209,17 +182,18 @@ module.exports = {
           }
           break;
         case "[":
+        case "{":
           node = this.node("offsetlookup");
```

Comment:
can you add a new `curly` property in order to keep track of offset type of `{` or `[`

Reply:
@ichiriac we have several places with `offsetlookup`, i want to refactor them and test before implement `curly` option (maybe some of them invalid), so let's do it in other PR

Reply:
ok, lets do it later - it was referenced to #380